File: HaskellFizzBuzz/fizzbuzz2.hs
divisible :: Int -> Int -> Bool
divisible x y = x `rem` y == 0

showInt :: Int -> String
showInt x | x `divisible` 15 = "FizzBuzz"
          | x `divisible` 3  = "Fizz"
          | x `divisible` 5  = "Buzz"
          | otherwise        = show x

fizzbuzz :: [String]
fizzbuzz = [showInt x | x <- [1..100]]

main = mapM_ putStrLn fizzbuzz

File: JuliaFizzBuzz/FizzBuzz2.jl
for i in 1:50
    i % 3 == 0 && i % 5 == 0 ? println(i, "FizzBuzz ") : 
    i % 3 == 0 ? println(i, "Fizz ") : 
    i % 5 == 0 ? println(i, "Buzz ") : 
    println(i)
end